### DISCUSSION TITLE
Predefined route can be stored?

### DISCUSSION THREAD

**Discussion by Cyrth** [organicmaps/organicmaps#7987] [Q&A]
Hello,
is it possible to store a predefined route in Organic Maps?
I can't drive the route that Maps suggests. I don't want to have to enter the new destination each time, when I jump from Point to Point.
Greetings

**Comment by biodranik** [comment]
You can import any routes in GPX or KML files. Why can't you drive the route? Can you please also provide more details about your issue to understand it better? Is the route wrong? Is it also wrong at OpenStreetMap.org?

**Reply by Cyrth** [reply]
Hi biodranik,
the Route which is chosen by Maps is possible. But I want to take my special route over the Alps. This not the shortest, but it is one with a lot spots i want to meet.

**Reply by biodranik** [reply]
You can add/mark key points on the map, and build a route to them when necessary as a workaround. Importing KML and GPX created in other apps and services should also help.

**Reply by Cyrth** [reply]
Thanks, I make a try
